css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,23 @@
+button {
+  font-size: large;
+  padding: 10px 20px 10px 20px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
+
+th:nth-child(even) {
+  background-color: black;
+  color: white;
+}
+
+th:nth-child(odd) {
+  background-color: black;
+  color: white;
+}
```

Comment:
If you're making the even and odd table headers the same, it's probably not worth splitting them up like this.